content/en/docs/configuration/acme/dns01/acme-dns.md

```diff
@@ -31,85 +30,141 @@ Information about setting up and configuring ACMEDNS is available on the
+    registration.
 
-> Note: that the "name" of the record always has the `_acme-challenge`
-> subdomain, and the "value" of the record matches exactly the `fulldomain`
-> field from registration.
+    At verification time, the domain name `d420c923-bbd7-4056-ab64-c3ca54c9b3cf.auth.example.com` will be a TXT
+    record that is set to your validation token. When the verifier queries `_acme-challenge.example.com`, it will
+    be directed to the correct location by this CNAME record. This proves that you control `example.com`
 
-At verification time, the domain name `d420c923-bbd7-4056-ab64-c3ca54c9b3cf.auth.example.com` will be a TXT
-record that is set to your validation token. When the verifier queries `_acme-challenge.example.com`, it will
-be directed to the correct location by this CNAME record. This proves that you control `example.com`
+4. Create a secret from the credentials JSON that was saved in step 2, this
+   secret is referenced in the `accountSecretRef` field of your DNS01
+   issuer settings. When creating an `Issuer` both this `Issuer` and
+   `Secret` must be in the same namespace. However for a `ClusterIssuer`
+   (which does not have a namespace) the `Secret` must be placed in the
+   same namespace as where the cert-manager pod is running in (in the
+   default setup `cert-manager`).
 
-4. Create a secret from the credentials JSON that was saved in step 2, this secret is referenced
-   in the `accountSecretRef` field of your DNS01 issuer settings.
-   When creating an `Issuer` both this `Issuer` and `Secret` must be in the same namespace.
-   However for a `ClusterIssuer` (which does not have a namespace) the `Secret` must be placed in
-   the same namespace as where the cert-manager pod is running in (in the default setup `cert-manager`).
+   ```sh
+   kubectl create secret generic acme-dns --from-file acmedns.json
+   ```
 
-```bash
-$ kubectl create secret generic acme-dns --from-file acmedns.json
+## Limitation of the `acme-dns` server
+
+The [`acme-dns`](https://github.com/joohoi/acme-dns) server has a [known
+limitation](https://github.com/jetstack/cert-manager/issues/3610): the same
+subdomain cannot be used multiple times in the same certificate. For
+example, the following certificate requires two separate challenges that
+will be issued simultaneously using the `acmeDNS` solver:
+
+```yaml
+kind: Issuer
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: my-acme-dns
+spec:
+  acme:
+    solvers:
+      - dns01:
+          acmeDNS:
+            accountSecretRef:
+              name: my-acme-dns-secret
+            host: my-acme-dns.com
+---
+kind: Certificate
+spec:
+  issuerRef:
+    name: issuer-1
+  dnsNames:
+    - "example.com"
+    - "*.example.com"
 ```
+
+The two DNS names above will generate two challenges, and one of the two
+challenges will "roll over" the other in a non deterministic way. This
+limitation comes from a "feature" mentioned in `acme-dns`'s README:
```

Comment:
I think https://github.com/joohoi/acme-dns/issues/76 might be better documentation?

Reply:
Done in c31897cc2b199ad4a956e178482c8e119a1b1f3a.